### ISSUE TITLE
Collapsible menu does not announce when it's expanded/collapsed to screenreader users

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#43]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[Aan de slag met API's](https://don.apps.digilab.network/kennisbank/apis/aan-de-slag/)

**Probleembeschrijving:**
De uitklapbare menu's in het hamburger menu kondigen niet aan wanneer ze worden geopend of gesloten, ook al hebben ze een aria-expanded attribuut. De knop mist een aria-controls attribuut om het te koppelen aan het paneel dat het bestuurt. De knop zou moeten verwijzen naar de ID van het paneel. Het paneel mist een id attribuut om bestuurd te worden door de knop. Het paneel heeft geen expliciete ARIA-rol om het te identificeren als een navigatie-element. De screenreader weet niet wat er wordt uitgeklapt/ingeklapt.

**Referentiecode:**
`<div class="navbaritems"><button aria-label="Navigatiebalk schakelen" aria-expanded="false" class="navbartoggle clean-btn" type="button">`

**Gerelateerde WCAG:**
WCAG 4.1.2 Name, Role, Value

**Referentieafbeelding:**
![A collapsible menu does not communicate to screenreader users its state correctly](https://github.com/user-attachments/assets/408f80b2-ad0a-43b2-b809-097159769b5a)

_**Referentienummer:** A032_
